CHANGELOG.adoc

```diff
@@ -0,0 +1,556 @@
+[[CopyArtifactPlugin-ChangeLog]]
+== Change Log
+
+:sectanchors:
+
+[[CopyArtifactPlugin-Version1.43]]
+=== Version 1.43 (Oct 5, 2019)
+
+* Support `+buildSelector+` in pipeline parameters
+(https://github.com/jenkinsci/copyartifact-plugin/pull/114[#114])
+
+[[CopyArtifactPlugin-Version1.42.1]]
+=== Version 1.42.1 (May 25, 2019)
```

Comment:
I usually recommend another format for changelogs so that tools like Dependabot can comprehend them. Not a big deal for historical changelogs

```suggestion
=== 1.42.1 

Release date: (May 25, 2019)
```

Reply:
Updated. Thanks.